services/distribution/src/main/java/app/coronawarn/server/services/distribution/dgc/client/ProdDigitalCovidCertificateClient.java

```diff
@@ -82,6 +82,14 @@ public BusinessRule getCountryRuleByHash(String country, String hash) throws Fet
         FetchBusinessRulesException::new);
   }
 
+  @Override
+  public BusinessRule getBnRuleByHash(String country, String hash) throws FetchBusinessRulesException {
```

Comment:
country parameter is not used. should be removed both here and from the interface.